experimental/framework/get-container/src/tinyliciousService.ts

```diff
@@ -7,19 +7,51 @@ import {
     IContainer,
     IRuntimeFactory,
 } from "@fluidframework/container-definitions";
-import { IDocumentServiceFactory, IUrlResolver } from "@fluidframework/driver-definitions";
+import { Container } from "@fluidframework/container-loader";
+import {
+    IDocumentServiceFactory,
+    IUrlResolver,
+} from "@fluidframework/driver-definitions";
 import { RouterliciousDocumentServiceFactory } from "@fluidframework/routerlicious-driver";
-import { InsecureTinyliciousTokenProvider, InsecureTinyliciousUrlResolver } from "@fluidframework/tinylicious-driver";
+import {
+    InsecureTinyliciousTokenProvider,
+    InsecureTinyliciousUrlResolver,
+} from "@fluidframework/tinylicious-driver";
 import { getContainer, IGetContainerService } from "./getContainer";
 
+export interface ITinyliciousFileConfig {
```

Comment:
I really don't like File here. 

So the idea is that we have two configs. Once defines the service specific container properties. The other defines the container itself. 

Maybe `serviceConfig` and `objectConfig` ?  or `serviceConfig` and `schemaConfig`?

Also we might want to separate create and get? maybe... That would mess with the generic though...

Reply:
Updated to be serviceConfig & objectConfig.
As for separation of create & get, OdspService actually expects two different interfaces for each respectively, where the create takes additional file metadata such as the its drive info and folder name but the get requires the URL to the file. It just so happens that the serviceConfig for Tinylicious is the same for both cases